docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.9'
+
+services:
+  web:
+    build: .
+    volumes:
+      - .:/app
+    ports:
+      - "8000:8000"
+    depends_on:
+      - db
+    command: >
+      sh -c "cd forum &&
+             python manage.py migrate &&
+             python manage.py runserver 0.0.0.0:8000"
```

Comment:
Avoid running migrations directly in the command. It's better to separate concerns using `entrypoint.sh`.
```
command: sh /app/entrypoint.sh
```
And in `entrypoint.sh`:
```
#!/bin/sh

echo "Applying migrations..."
python manage.py migrate

echo "Collecting static files..."
python manage.py collectstatic --noinput

echo "Starting server..."
python manage.py runserver 0.0.0.0:8000
```